js/exporter/jspdf/v3/options.js

```diff
@@ -0,0 +1,32 @@
+import { isDefined, isNumeric } from '../../../core/utils/type';
+
+function normalizeOptions(rows) {
```

Comment:
Maybe it's better to rename this method to normalizePaddings?
Currently it normalize paddings only. But it is discussable

Reply:
It is assumed that this method will validate not only padding object but also other parameters ( in future ), for example, color:

**types**  - 'string' | number | [number,number,number]
**values** - '#ff0000' | 128 | [128,128,128]

http://raw.githack.com/MrRio/jsPDF/master/docs/jsPDF.html#setTextColor

This is the reason why I named the methods like this:

```
function normalizeOptions(rows) {
    rows.forEach(row => {
        row.cells.forEach(({ pdfCell }) => {
            normalizePadding(pdfCell);
            // TODO: normalizeColor(pdfCell);
            // TODO: normalize some other parameter in the cell
        });
    });
}
```